python/apps/backend/backend/auth/providers.py

```diff
@@ -229,20 +229,22 @@ async def get_user(self, auth_token: ExternalAuthToken) -> ExternalUser:
 
 		# Request user's information from Azure
 		userinfo_response = requests.get(
-			# Currently userinfo_endpoint only returns "sub". We need to use /v1.0/users for other info
-			"https://graph.microsoft.com/v1.0/users",
+			# Currently userinfo_endpoint only returns "sub". We need to use /v1.0/me for other info
+			"https://graph.microsoft.com/v1.0/me",
```

Comment:
Hmm the /users endpoint is still working for me. I wonder whether I changed some configuration for the unprivileged users in the Azure options. I might be totally wrong though.. Might have only tested it with a privileged user..

Thanks for pointing this out!